SlugifierInterface.php

```diff
@@ -9,21 +9,31 @@
  * file that was distributed with this source code.
  */
 
+namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
+
+if (!interface_exists('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface')) {
+    /**
+     * @deprecated Since 1.0, to be removed in 2.0.
+     */
+    interface SlugifierInterface {
```

Comment:
@dbu because of the lack of response from lukas, let's take your solution to get this merged soon (so we can release CoreBundle 1.3.0-RC1 as well, meaning almost all important CMF packages have a RC1 release).

However, what should we say in the deprecation message? "To be removed in 2.0", releasing a 2.0 together with other 2.0 versions of the CMF?